media-libs/embree/embree-4.3.1.ebuild

```diff
@@ -121,3 +117,30 @@ src_configure() {
 
 	cmake_src_configure
 }
+
+src_test() {
+	local CMAKE_SKIP_TESTS=(
```

Comment:
I would just add a comment here stating that these test failures is because of the required `EMBREE_BACKFACE_CULLING` options for Blender Cycles.